lib/services/github.go

```diff
@@ -26,7 +26,11 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
-var ErrRequiresEnterprise = trace.AccessDenied("this feature requires Teleport Enterprise")
+// generic error for attempting usage of a enterprise feature
```

Comment:
godoc comments should typically start with the name of the identifier they are documenting

```suggestion
// ErrRequiresEnterprise indicates that a feature requires
// Teleport Enterprise
```